gpflow/utilities/bijectors.py

```diff
@@ -5,17 +5,50 @@
 from .. import config
 from .utilities import to_default_float
 
+
 __all__ = ["positive", "triangular"]
 
 
-def positive(lower: Optional[float] = None):
-    lower_value = config.default_positive_minimum()
-    if lower_value is None:
-        return tfp.bijectors.Softplus()
+_POSITIVE_BIJECTOR_MAP = {
+    "exp": tfp.bijectors.Exp,
+    "softplus": tfp.bijectors.Softplus,
+}
+
+
+class Shift(tfp.bijectors.AffineScalar):
```

Comment:
* Check the comment in the test about printing.
* I believe we don't need this custom class

Reply:
I agree this custom class _might_ be overkill...but the benefit of an informative print string (especially as this will become the new default) is compelling.

Reply:
Resolving this conversation - can move discussion to `test_printing`